build/wix/main.wxs

```diff
@@ -157,6 +157,18 @@
       <ComponentRef Id="PathUser" />
       <ComponentRef Id="PathSystem" />
     </Feature>
+    <UI>
+       <UIRef Id="WixUI_RD" />
+       <Publish Dialog="ExitDialog"
```

Comment:
We have a bunch of overrides in `dialogs.wxs`; this should probably live there?
https://github.com/rancher-sandbox/rancher-desktop/blob/0a26306847d0afedb2f0c98cff02c21149159892/build/wix/dialogs.wxs

Reply:
The docs I found ( https://documentation.help/WiX-Toolset/run_program_after_install.html ) state that the `ExitDialog` code has to be in a `Product` element, and we only have one, in `main.wxs`.

Reply:
Inside the `<Product>` we have a `<UIRef>`, so the `<UI>` from `dialogs.wxs` is basically included there, so things are fine.

Reply:
ok, that works